src/main/java/com/google/cloud/spanner/pgadapter/statements/CopyStatement.java

```diff
@@ -169,11 +169,11 @@ private void queryInformationSchema() throws SQLException {
     Map<String, TypeCode> tableColumns = new LinkedHashMap<>();
     PreparedStatement statement =
         this.connection.prepareStatement(
-            "SELECT "
+            "/*GSQL*/SELECT "
```

Comment:
why do we need to issue this as a GoogleSQL query?

Reply:
The backend no longer returns these as GoogleSQL types for this query. It instead returns the PG types now. We need the GoogleSQL types to fill out the mutations.

This was just recently changed:
https://buganizer.corp.google.com/issues/198312860

Reply:
I see. Can you add a comment?

// The mutation API requires GoogleSQL type names instead of PostgreSQL type names for the table column types
// Issue the information_schema table query as a GoogleSQL query so that it will return GoogleSQL type names.

Reply:
Done.